ros2interface/test/test_cli.py

```diff
@@ -200,6 +200,22 @@ def test_package_on_test_msgs(self):
         )
         assert all(interface in output_lines for interface in some_interfaces)
 
+    def test_package_on_test_msgs_only_srvs(self):
+        with self.launch_interface_command(
+            arguments=['package', 'test_msgs', '-s']
+        ) as interface_command:
+            assert interface_command.wait_for_shutdown(timeout=2)
+        assert interface_command.exit_code == launch_testing.asserts.EXIT_OK
+        output_lines = interface_command.output.splitlines()
+        assert launch_testing.tools.expect_output(
+            expected_lines=itertools.repeat(
+                re.compile(r'test_msgs/srv/[A-z0-9_]+'), len(output_lines)
+            ),
+            lines=output_lines,
+            strict=True
+        )
+        assert all(interface in output_lines for interface in some_services_from_test_msgs)
```

Comment:
It is probably overkill, but I'd feel a little better about this if we had 3 tests, one for each of `-s`, `-m`, and `-a`.  Also, I'm concerned about a 2 second timeout for the shutdown, particularly on Windows; I'll suggest 5 seconds instead.